apps/kibana/vars/vault/main.yml.j2

```diff
@@ -0,0 +1,7 @@
+# customer: {{ customer }}
+# env_type: {{ env_type }}
+
+# kibana
+XPACK_SECURITY_ENCRYPTIONKEY: {{ lookup('password', '/dev/null length=50') }}
+XPACK_REPORTING_ENCRYPTIONKEY: {{ lookup('password', '/dev/null length=50') }}
+XPACK_ENCRYPTEDSAVEDOBJECTS_ENCRYPTIONKEY: {{ lookup('password', '/dev/null length=50') }}
```

Comment:
From what I understand, you provides random generated passwords as default values for these secrets.
The goal is that each customer define its own secrets in a vault file.

But if you forget to define these for a customer, these default values will be applied silently, with no warning.
And a side effect that I see is that the secrets will change each time the playbook is applied.

According to the [ansible documentation ](https://docs.ansible.com/ansible/latest/plugins/lookup/password.html): 
> A special case is using /dev/null as a path. The password lookup will generate **a new random password each time**

Reply:
This file is a vault template used once while generating the application's vault for your project using the `create_vaults.yml` playbook. When the vault has been generated by random generated passwords, it can be edited using the `ansible-vault` tool but it won't be changed when generating secrets.

Reply:
Ok, I missed the fact that it is used only in the`create_vaults.yml` playbook !
Thanks for the clarifications :+1: